frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -13,22 +13,9 @@
         </v-col>
         <v-col cols="1" />
       </v-row>
-      <div v-for="section in contentNode.sections().items" :key="section._meta.self">
-        <!-- add before -->
-        <v-row no-gutters class="row-inter" justify="center">
-          <v-col cols="1">
-            <v-btn v-if="!layoutMode"
-                   icon
-                   small
-                   class="button-add"
-                   color="success"
-                   @click="addSection">
-              <v-icon>mdi-plus</v-icon>
-            </v-btn>
-          </v-col>
-        </v-row>
 
-        <api-form :entity="section">
+      <api-sortable v-slot="sortable" :collection="sections" :collection-uri="'/content-type/storyboards?contentNodeId=' + contentNode.id">
```

Comment:
Siehe https://github.com/ecamp/ecamp3/blob/03bbbb7bfbe5c008a3f38b583e64c6811e7183b1/backend/module/eCampCore/src/Hydrator/ActivityHydrator.php#L61-L68 für eine existierende Lösung wie man eine Embedded Collection mit einem self-Link ausstatten kann.

Reply:
Yep, cool, dass das nun geht. Link hatte ich in der Strategy bereits schon vorbereit gehabt, aber hatte damals irgendwie noch nicht funktioniert. So können wir hoffentlich auch unseren API Root etwas entmüllen.